simulator/src/test/java/com/hazelcast/simulator/agent/JavaInstallationExceptionTest.java

```diff
@@ -1,21 +0,0 @@
-package com.hazelcast.simulator.agent;
-
-import org.junit.Test;
-
-import static org.junit.Assert.assertEquals;
-
-public class JavaInstallationExceptionTest {
```

Comment:
Don't waste time testing exceptions! They will be tested by the logic throwing them. And if a constructor is not used, delete it!